### ISSUE TITLE
Include a Code of Conduct doc

### ISSUE THREAD

**Issue by mpcnat** [productof/brand#3]
Create a Code of Conduct document that represents all chapters, or can be used as a template for individual chapters as they come on board.

Look at Phill Farrugia's version as a basis https://github.com/phillfarrugia/product-of-sfo/blob/master/CODE_OF_CONDUCT.md

**Comment by lukebrooker** [comment]
Yep. I have this in a site map for product of website too. Not sure which repo it should be in. Here would be ok for now.

**Comment by phillfarrugia** [comment]
I used a template to generate that code of conduct, while we haven't had any issues in the Product Of SFO community with it, we're still in our early days and haven't had to whip it out or refer to it for any specific instances or potential violations of it.

There are definitely far more experienced community organizers that can provide input/improvements to it.
